pandas/core/ops/__init__.py

```diff
@@ -609,8 +609,25 @@ def _combine_series_frame(left, right, func, axis: int):
     return left._construct_result(new_data)
 
 
-def _align_method_FRAME(left, right, axis):
-    """ convert rhs to meet lhs dims if input is list, tuple or np.ndarray """
+def _align_method_FRAME(left, right, axis, flex: Optional[bool] = False, level: Optional[Label] = None):
```

Comment:
```suggestion
def _align_method_FRAME(left, right, axis, flex: Optional[bool] = False, level: Optional[Level] = None):
```

Sorry typo on previous; we actually have Level defined in pandas._typing already (probably need to import)